### ISSUE TITLE
[BUG] Babel aliases do not work in non-Babel environments

### ISSUE THREAD

**Issue by rassie** [pankod/refine#1505]
**Describe the bug**

Refine defines several Babel aliases in its distribution in `.babelrc`: 
```
{
    "plugins": [
        [
            "module-resolver",
            {
                "root": "./",
                "alias": {
                    "@components": "./src/components",
                    "@containers": "./src/containers",
                    "@pages": "./src/pages",
                    "@contexts": "./src/contexts",
                    "@dataProviders": "./src/dataProviders",
                    "@hooks": "./src/hooks",
                    "@test": "./test",
                    "@definitions": "./src/definitions"
                }
            }
        ]
    ]
}
```

These aliases do not get resolved in any non-Babel environment, like [Vite](https://vitejs.dev/), leading to errors.

**To Reproduce**
1. Initialize a new Vite project with React and TypeScript with `npm init vite@latest` (https://vitejs.dev/guide/#scaffolding-your-first-vite-project).
2. Follow first steps on Refine tutorial (https://refine.dev/docs/) until just before "Adding resources" chapter. 
3. Run `npm run build`
4. Observe errors in the console like `ERROR(TypeScript)  Cannot find module '@components' or its corresponding type declarations.`

**Expected behavior**

Should work with Vite just like it does with `create-react-app`.

**Comment by omeraplak** [comment]
Hey @rassie ,
Thanks a lot for the issue. I'll see what we can do!

**Comment by salihozdemir** [comment]
Hey @rassie,
We released v3.2.4! You can check out refine vite example in [here](https://github.com/pankod/refine/tree/master/examples/vite). If you have a problem your project, feel free to write.

**Comment by rassie** [comment]
Thanks! Will be testing in the next couple of days.
